libs9s/s9srpcclient.cpp

```diff
@@ -10420,6 +10421,13 @@ S9sRpcClient::getDbVersions(
     request["vendor"] = vendor;
     if(patchNumber)
         request["patch_number"] = true;
+    if(useVendorApi)
+    {
+        if(patchNumber)
+            request["use_vendor_api"] = true;
+        else
+            ::printf("--use-vendor-api is only supported with --list-3d. Skipping option\n");
+    }
```

Comment:
I am confused. `--list-3d` (in the printf call) supposed to be a list, `patchNumber` supposed to be a number and not a list.
How they are related to each other?

Reply:
list-3d refers to the list of versions with 3 digits and patchNumber is the 3rd digit when using 3d versions instead of normal 1-2d versions. 3d versions = 'major.minor.patch'